Dockerfile

```diff
@@ -284,10 +257,8 @@ COPY --chown=airflow:airflow airflow/bin/airflow ${AIRFLOW_SOURCES}/airflow/bin/
 
 # The goal of this line is to install the dependencies from the most current setup.py from sources
 # This will be usually incremental small set of packages in CI optimized build, so it will be very fast
-# In non-CI optimized build this will install all dependencies before installing sources.
```

Comment:
What has happened to the CI-optimizations? Will we need a second Dockerfile for ci builds?

Reply:
It is still there - controlled by AIRFLOW_CONTAINER_CI_OPTIMISED_BUILD parameter. For prod image, there is only one pip install.